lib/stats/localtrackstats.js

```diff
@@ -21,19 +21,19 @@ class LocalTrackStats extends TrackStats {
       bytesSent: {
         value: typeof statsReport.bytesSent === 'number'
           ? statsReport.bytesSent
-          : null,
+          : 0,
```

Comment:
@makarandp0 I think we should retain `null` here (since this is public-facing) and convert them to 0 when we are actually publishing it.